src/Extensions/hcox_megan_mod.F90

```diff
@@ -3583,6 +3584,14 @@ SUBROUTINE HCOX_Megan_Init( HcoState, ExtName, ExtState, RC )
         CALL HCO_ERROR( 'ERROR 35', RC, THISLOC=LOC )
         RETURN
     ENDIF
+    Inst%InvMEGANAll = .FALSE.
+    CALL GetExtOpt( HcoState%Config, ExtNr, 'InvMEGAN All ', &
+                    OptValBool=Inst%InvMEGANAll, Found=FOUND, RC=RC )
+    IF ( RC /= HCO_SUCCESS ) THEN
+        CALL HCO_ERROR( 'ERROR 36', RC, THISLOC=LOC )
```

Comment:
Seems 36 is already taken. May have to find one that isn't.

https://github.com/geoschem/HEMCO/blob/05d388c2e557efc68a6e13222f710b565ae69b68/src/Extensions/hcox_megan_mod.F90#L3601

Reply:
Thanks Zach, I changed the number to one not taken.